bot/cogs/views/sticky.py

```diff
@@ -64,23 +61,20 @@ async def construct_pages(self, ctx: commands.Context, list_data: List[Dict[str,
                 )
 
                 for sticky_data_list in page_data_list:
-                    if sticky_data_list == page_data_list[1] and len(page_data_list[1]) == 0:
-                        continue
-
                     field_value = ""
                     field_name = ""
 
                     if sticky_data_list == page_data_list[0]:
-                        field_name = "Channel  |  Message"
+                        field_name = "Channel  Name"
                     else:
-                        field_name = "|"
+                        field_name = ""
 
                     for sticky_data in sticky_data_list:
-                        if len(sticky_data["message"]) > 25:
-                            message = sticky_data["message"][:20] + "..."
+                        if len(sticky_data["message"]) > 250:
+                            message = sticky_data["message"][:250] + "..."
                         else:
                             message = sticky_data["message"]
-                        row_string = f"<#{sticky_data['channel_id']}> message={message}\n"
+                        row_string = f"<#{sticky_data['channel_id']}> \n > {message} \n\n"
```

Comment:
![image](https://user-images.githubusercontent.com/20255031/220012843-f4f87f92-f92d-497c-a9e0-1d155bd8bc5e.png)

This is failed when the message contains multiple lines.